lib/shadow.ml

```diff
@@ -126,8 +125,8 @@ let to_string p =
     p.flag
 
 let db_to_string db = db
-  |> List.map to_string
-  |> String.concat "\n"
+                      |> List.map to_string
```

Comment:
It doesn't like the `db` after the `=` in the function declaration.   If I add a newline after the `=` it does something more sensible.